Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -310,6 +318,18 @@ open class ChatMessageListVC:
 
         cell.messageContentView?.delegate = self
         cell.messageContentView?.content = message
+        cell.dateSeparatorView.isHidden = true
+
+        let previousIndexPath = IndexPath(row: indexPath.row + 1, section: indexPath.section)
+        if let previousMessage = dataSource?.chatMessageListVC(self, messageAt: previousIndexPath),
+           let currentMessage = message,
+           Self.isDateSeparatorEnabled {
```

Comment:
Self.isDateSeparatorEnabled should be the first check, so that if it is false, you don't access the data source

Reply:
Yes that is true 👍 I did It in this way because it looks cleaner 😃 And I thought that the efficiency here in this case might not be a big deal. But I'll change it anyway 👍

Reply:
✅